test/rtcrtpsender.js

```diff
@@ -43,45 +42,45 @@ var sdp = [
   'a=ssrc:2 msid:stream 456'
 ].join('\r\n') + '\r\n';
 
-tape('.addTrack(track, stream)', function(t) {
+tape('.addTrack(track, [stream])', function(t) {
```

Comment:
Per comment above, using rest parameter syntax, there should be no change necessary here. Can you please revert all such changes below?